test/mocks/crds/testing.solo.io_v1_MockResource.yaml

```diff
@@ -57,15 +57,23 @@ spec:
               type: string
             oneofTwo:
               type: boolean
+            reporterStatus:
+              properties:
+                statuses:
+                  additionalProperties:
+                    type: object
+                    x-kubernetes-preserve-unknown-fields: true
+                  description: Mapping from namespace to the Status written by the
+                    controller running in that namespace.
+                  type: object
+              type: object
             someDumbField:
               type: string
             status:
-              description: The status field comment
```

Comment:
We have a status field on the spec, and a status resource on the CRD. What is the difference between the two? They seem to be generated from the same field, why is there not a duplicate reporterStatus as well?

Reply:
I don't know a whole lot about these CRD schemas, but my guess is it's because status and namespacedStatuses are in a oneof.  So the 'status' keys in the schema now refer to a oneof value rather than a `Status` value.